container_test.go

```diff
@@ -107,6 +107,10 @@ func TestNewContainer(t *testing.T) {
 func TestContainerStart(t *testing.T) {
 	t.Parallel()
 
+	if runtime.GOOS == "darwin" {
+		t.Skip("test with /bin/sh isn't supported on darwin")
```

Comment:
Why? Is this related to tty stuff?

Reply:
Tty is also missing, in addition to the same reason to the above comment.